### ISSUE TITLE
precompute这个参数的注释所产生的疑问

### ISSUE THREAD

**Issue by huhuhang** [shiyanlou/louplus-ml#104]
- 问题链接：https://www.shiyanlou.com/questions/108572
- 学员昵称：书山压力大EE
- 提问时间：2019-04-27T14:06:31.000Z

---

precompute这个参数的注释所产生的疑问:     为什么用预先计算 的Gram 矩阵 可以加速计算,   数学原理是什么?

<pre style="font-size:14px; line-height:17px;" class="hljs">
sklearn.linear_model.Lasso(alpha=1.0, fit_intercept=True, normalize=False, precompute=False, copy_X=True, max_iter=1000, tol=0.0001, warm_start=False, positive=False, random_state=None, selection='cyclic')
</pre>
- `alpha`: 正则化强度，默认为 1.0。
- `fit_intercept`: 默认为 True，计算截距项。
- `normalize`: 默认为 False，不针对数据进行标准化处理。
- `precompute`: 是否使用预先计算的 Gram 矩阵来加速计算。
- ...

**Comment by songyer** [comment]
已回答
